machine/v1/types_nutanixprovider.go

```diff
@@ -78,6 +78,17 @@ type NutanixMachineProviderConfig struct {
 	// +optional
 	Categories []NutanixCategory `json:"categories"`
 
+	// gpus is a list of GPU devices to attach to the machine's VM.
+	// +listType=map
+	// +listMapKey=type
```

Comment:
let's not use type listType map here. type being listMapKey effectively forces 1 GPU which is not what we want.

Reply:
fixed